syntax = "proto3";

package Greeter;

service GreeterService {
    rpc SayHi (HiRequest) returns (HiResponse) {}
    rpc SayHiAgain (HiRequest) returns (HiResponse) {}
}

message HiRequest {
    string target_name = 1;
}

message HiResponse {
    string message = 1;
}
